apis/logging/v1/output_types.go

```diff
@@ -282,3 +285,28 @@ type Splunk struct {
 	// +optional
 	Fields []string `json:"fields,omitempty"`
 }
+
+// Http provided configuration for sending json encoded logs to a generic http endpoint.
+type Http struct {
+	// Headers specify optional headers to be sent with the request
+	// +optional
+	Headers map[string]string `json:"headers,omitempty"`
+
+	// Timeout specifies the Http request timeout in seconds. If not set, 10secs is used.
+	// +optional
+	Timeout string `json:"timeout,omitempty"`
+
+	// Method specifies the Http method to be used for sending logs. If not set, 'POST' is used.
+	// +kubebuilder:validation:Enum:=GET;HEAD;POST;PUT;DELETE;OPTIONS;TRACE;PATCH
+	// +optional
+	Method string `json:"method,omitempty"`
+
+	// MaxBatch specifies maximum size of batch in bytes
+	// +optional
+	MaxBatch string `json:"maxBatch,omitempty"`
```

Comment:
@vimalk78 this is the param we discussed that is not supported in both collectors?  If so please remove for the time being.  cc @alanconway

Reply:
removed